src/sgame/sg_bot_parse.cpp

```diff
@@ -164,6 +164,7 @@ static AIValue_t Ratio( int current, int max, bool infiniteAmmo )
 static AIValue_t percentAmmoClip( gentity_t *self, const AIValue_t* )
 {
 	ASSERT( self && self->client );
+	ASSERT( wpa->maxAmmo > 0 );
```

Comment:
Assertion seems wrong. I think maxAmmo is 0 for infinite ammo weapons

Reply:
wrong. Adv goon have maxAmmo = 3 for barbs, which are also infinite.